<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>250</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>SettingsDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="languageText">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="text">
         <string>Language</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>142</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="languageComboBox">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chinese</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="autoRunOnStartUpCb">
        <property name="text">
         <string>Auto run on start up</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="keepDialogOnErrorOccurredCb">
        <property name="text">
         <string>Keep dialog display on error occurred</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="removeToTrashCb">
        <property name="text">
         <string>Remove files to trash</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="symlinkHotkeyText">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="text">
            <string>Symlink hotkey</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KeyCombinationInputer" name="symlinkHotkeyInputer"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_5" native="true">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="hardlinkHotkeyText">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="text">
            <string>Hardlink hotkey</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>46</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KeyCombinationInputer" name="hardlinkHotkeyInputer"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyCombinationInputer</class>
   <extends>QLineEdit</extends>
   <header>keycombination_inputer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
